### DISCUSSION TITLE
Import error

### DISCUSSION THREAD

**Discussion by Zaheer-10** [explosion/spaCy#12745] [Help: Coding & Implementations] [answered]
NotImplementedError: A UTF-8 locale is required. Got ANSI_X3.4-1968 this is when trying to install !pip install neuralcoref

**Comment by kadarakos** [comment, marked as answer]
Unfortunately this seems to be a known cuda bug: https://github.com/explosion/spaCy/issues/11909.
As a side note you could consider using the `spacy-experimental` coreference resolver -- which you can read about [here](https://explosion.ai/blog/coref) -- or [fastcoref](https://github.com/shon-otmazgin/fastcoref#spacy-component).
